src/AffectedUserFlow/CodeInput/CodeInputScreen.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react"
+import { cleanup, render } from "@testing-library/react-native"
+import "@testing-library/jest-native/extend-expect"
+
+import CodeInputScreen from "./CodeInputScreen"
+import { AffectedUserProvider } from "../AffectedUserContext"
+import {
+  PermissionsContext,
+  ENPermissionStatus,
+} from "../../PermissionsContext"
+import { PermissionStatus } from "../../permissionStatus"
+
+afterEach(cleanup)
```

Comment:
I would hope 🤞  that this gets run automatically if it's not the case we might have something going on on our setup